### PR TITLE
[Az.RecoveryServices.Backup] Added MUA support for V1 vaults

### PR DESCRIPTION
**Pull request by hiaga**
## Description

Added MUA support for V1 vaults

## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [x] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {[[Azure Backup] Adding support for multi-user authorization for Recovery Services vaults](https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/1201)}
    - [x] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/RecoveryServices/RecoveryServices/ChangeLog.md [hunk 1]
```diff
@@ -18,6 +18,7 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
+8 Added support for Multi-user authorization using Resource Guard for Recovery sevrices vault.
```
Comment (wyunchi-ms) [inline src/RecoveryServices/RecoveryServices/ChangeLog.md:21]:
```suggestion
* Added support for Multi-user authorization using Resource Guard for Recovery sevrices vault.
```

**Comment by wyunchi-ms** [comment]
Since you update the SDK version, you may need to re-record the test cases.

File: src/RecoveryServices/RecoveryServices/ChangeLog.md (hunk 1 above)
Comment (hiaga) [reply src/RecoveryServices/RecoveryServices/ChangeLog.md:21]: typo, will update

**Review by wyunchi-ms — APPROVED** [review]

**Review by wyunchi-ms — APPROVED** [review]

**Pull request merged**